pypesto/ensemble/ensemble.py

```diff
@@ -616,14 +618,26 @@ def from_optimization_endpoints(
             A pyPESTO result that contains an optimization result.
         cutoff:
             Exclude parameters from the optimization if the
-            nllh is higher than the `cutoff`.
+            nllh is higher than the `cutoff`. If cutoff is None, a cutoff
+            will be calculated based on the percentile passed.
         max_size:
             The maximum size the ensemble should be.
+        percentile:
+            Percentile of a chi^2 distribution. Used to determinie the
+            cutoff value.
 
         Returns
         -------
         The ensemble.
         """
+        if cutoff is None:
+            if percentile is None:
+                raise ValueError(
+                    'Neither cutoff nor percentile were set. '
+                    'Determining a cutoff value is not '
+                    'possible. Please set one of the two.'
+                )
+            cutoff = calculate_cutoff(result=result, percentile=percentile)
```

Comment:
Warn if percentile is not default, and both percentile and cutoff are specified? Another reason why defaulting both to `None` can be better